cmd/wordrow/wordmap.go

```diff
@@ -104,7 +104,12 @@ func processInlineMapping(mapping string, wordmap *wordmaps.WordMap) error {
 		return errors.Newf("Invalid CLI defined mapping '%s'", mapping)
 	}
 
-	wordmap.AddOne(values[0], values[1])
+	from, to := values[0], values[1]
+	if stringsx.IsEmpty(from) || stringsx.IsEmpty(to) {
+		return errors.Newf("Invalid CLI defined mapping '%s'", mapping)
```

Comment:
This line is identical to line 104 (see :arrow_down:) ideally this is refactor to reduce duplication.

https://github.com/ericcornelissen/wordrow/blob/22b2491ac98cf1e281b8d625ac0a17862332aed3/cmd/wordrow/wordmap.go#L104